damus.xcodeproj/project.pbxproj

```diff
@@ -1035,6 +1040,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```

Comment:
Does this show warnings for non-localized strings?

Reply:
I haven't figured out how to trigger it. I tried removing one of the NSLocalizedString macros but I didn't see any warnings.

Reply:
Yeah, I've had the same experience. Can't hurt though if it starts working eventually.